network.py

```diff
@@ -19,6 +19,7 @@ def connect(self):
             self.client.connect(self.addr)
             # Loads byte data
             return pickle.loads(self.client.recv(BYTE_SIZE))
+            
```

Comment:
BIG CHANGE HERE! ABSOLUTELY CRUCIAL DO ***NOT*** DELETE. EVER! (will forget and delete later maybe)